### ISSUE TITLE
How to run elasticmq-rest with akka-http-2.4.x

### ISSUE THREAD

**Issue by harmeetsingh0013** [adamw/elasticmq#77]
I am creating a project using akka-http-2.4.8 and using amazon-sqs. For sqs functionality testing, i am using elasticmq-rest. I am using `"org.elasticmq" %% "elasticmq-rest-sqs" % "0.9.3"` dependency. But when i run the test cases, i got following exception: 

```
java.lang.NoClassDefFoundError: akka/http/scaladsl/server/directives/WebsocketDirectives

	at org.elasticmq.rest.sqs.TheSQSRestServerBuilder.start(SQSRestServerBuilder.scala:96)
   -------------
```

This exception might-be version conflict. I am also exclude dependency from elastic-mq but still got an exception. How to resolve this issue?

**Comment by adamw** [comment]
Probably the dependencies in ElasticMQ would need updating. Could you try doing that and maybe submit a PR?

**Comment by adamw** [comment]
Fixed in 0.10.0
